src/main/java/org/example/models/JobRoleResponse.java

```diff
@@ -0,0 +1,72 @@
+package org.example.models;
+
+import java.util.Date;
+
+public class JobRoleResponse {
+    private int roleId;
+    private String roleName;
+    private String locations;
+    private String capability;
+    private String band;
+    private Date closingDate;
+
+    public JobRoleResponse(final int roleId, final String roleName,
+                   final String locations,
```

Comment:
Should not be plural as it's a string (should be an enum)

Reply:
its a string of enums